lib/teleterm/clusters/cluster_auth.go

```diff
@@ -40,10 +41,16 @@ import (
 
 // SyncAuthPreference fetches Teleport auth preferences and stores it in the cluster profile
 func (c *Cluster) SyncAuthPreference(ctx context.Context) (*webclient.WebConfigAuthSettings, error) {
-	_, err := c.clusterClient.Ping(ctx)
+	pingResponse, err := c.clusterClient.Ping(ctx)
 	if err != nil {
 		return nil, trace.Wrap(err)
 	}
+	pingResponseJSON, err := json.Marshal(pingResponse)
+	if err != nil {
+		c.Log.WithError(err).Debugln("Could not marshal ping response to JSON")
```

Comment:
It might be useful to log malformed (i.e. non-JSON) response.

Reply:
`fmt.Sprintf("%+v", pingResponse)` follows pointers just one level deep, so it won't able to log the full response, but it should be good enough.

Since the ping response itself is fetched as a JSON object, I think it won't happen often that we have trouble marshaling it back to JSON.

Reply:
Sorry, I confused myself here, I was sure we are unmarshalling the response here 🤦 I think your original code was perfectly fine, apologies for the noise.

Reply:
Ok, I'm going to remove that latest commit then!